pkgs/tools/security/masscan/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, libpcap }:
+
+stdenv.mkDerivation rec {
+  name = "masscan-${version}";
+  version = "2017-02-04";
```

Comment:
The last release is 3 years old and in the meantime many bugfixes and patches have been committed.